apps/pigment-css-vite-app/src/pages/material-ui/index.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { useLocation, matchRoutes, Link } from 'react-router-dom';
-import { css } from '@pigment-css/react';
+import Box from '@pigment-css/react/Box';
```

Comment:
I guess there will be a question about the import in the future comparing:

```js
import { css } from '@pigment-css/react';
import Box from '@pigment-css/react/Box';
```

I think we should pick one way of import.

Reply:
I'd prefer `import { Box } from '@pigment-css/react';` over default import.

Reply:
I'd prefer that everything is on a subpath. We don't want to repeat the same mistake of having barrel file exports from the start when we have control. Otherwise later, it'd harder to get the changes to be adopted by the user.

Reply:
Okay, let's use default export for Box. I added a task to move other APIs to the subpath before the stable beta release.